API.md

```diff
@@ -0,0 +1,49 @@
+<pre>Activities</pre>
+title | CharField  
+subtitle | CharField  
+description | TextField  
+image_url | UrlField  
+image_retina_url | UrlField  
+url | UrlField  
+difficulty | CharField (choices)  
+duration | CharField (choices)  
+**Skills** | *:*  
+**WebLitSkills** | *:*  
+**Topic** | 1:*  
+**LessonPlan** | *:*  
+**locale** | CharField (choices) (pycountry)  
+translation_of | ‘self’  
+
```

Comment:
Ah, I was going to ask that earlier but got distracted. We want to consider this in the scope of something like network pulse, so those two fields would likely be very useful.